__tests__/page.test.tsx

```diff
@@ -0,0 +1,159 @@
+import { render, screen, fireEvent, waitFor, act } from '@testing-library/react';
```

Comment:
[Frontend] Fixes: #13 Add Some Integration Testing

This is not very comprehensive but atleast introduces some integration testing to ensure main feature functionality is not broken as changes are continued to be made. This will help reduce the risk of regression, along with helping to support our CI/CD pipeline while encouraging more confidence in our deployments. 

Eventually, we should add some more integration tests in addition to unit and end to end tests.